resources/qml/CheckBoxWithTooltip.qml

```diff
@@ -28,11 +28,11 @@ CheckBox
         {
             anchors.verticalCenter: parent.verticalCenter
             anchors.horizontalCenter: parent.horizontalCenter
-            width: Math.round(parent.width / 2.5)
-            height: Math.round(parent.height / 2.5)
+            height: UM.Theme.getSize("checkbox_mark").height
```

Comment:
This component should not be of type Checkbox but of type UM.Checkbox (or even Cura.Checkbox!)

Right now there is quite a bit of code duplication.